README.rst

```diff
@@ -344,6 +344,16 @@ See `motohawk_no_floating_point_numbers.h`_ and
 `motohawk_no_floating_point_numbers.c`_ for the contents of the
 generated files.
 
+In this example we use ``--include_signal_names`` to generate
+``#define`` definitions for the CAN Frame Signal names.
+
+.. code-block:: text
+
+   $ python3 -m cantools generate_c_source --include_signal_names tests/files/dbc/motohawk.dbc
```

Comment:
that option should use minuses instead of underscores (`--include-signal-names`) and it should probably be called `--generate-signal-name-macros`. That said, I don't really see any harm in including these macros unconditionally, if it is decided that we want them...